### ISSUE TITLE
Inquiry on the Role of Translation in Dual Quaternions in Your Code

### ISSUE THREAD

**Issue by Lucifer-G0** [UPC-ViRVIG/SparsePoser#4]
Hello,

After reviewing your implementation, I noticed that dual quaternions, which are excellent for representing both rotation and translation, are utilized. However, I am a bit unclear about the role that translation plays in your code, especially since it does not seem to be explicitly used in the final results. Additionally, in my case, it appears that translation might not be necessary for the intermediate processes.

Could you please shed some light on why translation is included in the dual quaternion formulation in your code?

Understanding your rationale would be incredibly helpful for my project and could potentially guide some optimizations I am considering.

Thank you very much for your time and assistance.

**Comment by JLPM22** [comment]
Hi!

We found that providing translational information to the network improved the results. This is probably because it's more challenging to interpret rotations directly. Having said this, we are taking different approaches in newer projects that may help improve SparsePoser.

- We removed the translational part for all the decoder structure and output. While translations help during the encoding process, we do not use them in the end, and we can reduce the network size by removing them from the decoding process. 
- Neural networks can understand positions much faster and easier than rotations. We are experimenting with working directly with joint positions local to the root. However, to recover joint rotations from joint positions, we must run a traditional IK or a learned IK at the last step.

Hope it helps!
